.github/pull_request_template.md

```diff
@@ -0,0 +1,29 @@
+# Summary
+<!-- (Madantory) short summary of this PR. -->
+
+# Details
+<!-- the details in pr like some useful information -->
+
+# Reminders 
+<!-- for developers and reviewers because most of them are not checked automatically. 
+
+Ensure you have set up the pre-commit and the corresponding pre-commit hook.  
+Use `pre-commit run --from-ref origin/develop --to-ref HEAD` to ensure the branche changes compared to origin/develop are formatted through pre-commit.
```

Comment:
```suggestion
Use `pre-commit run --from-ref origin/develop --to-ref HEAD` to ensure the branch changes compared to origin/develop are formatted through pre-commit.
```

Reply:
Thanks! It is surprising to me that I have so dump typo here 🫢